src/i18n/pt-br.js

```diff
@@ -0,0 +1,445 @@
+            ok: "ENVIAR"
+        },
+        unlockConfirm: {
+            title: "Confirmar desbloqueamento",
+            ok: "DESBLOQUEAR"
+        },
+        unlockServiceNode: {
+            title: "Desbloquear nódulo de serviço",
+            confirmTitle: "Confirmar desbloqueio",
+            message: "Deseja desbloquear o nódulo de serviço?",
+            ok: "DESBLOQUEAR"
+        }
+    },
+    fieldLabels: {
+        // Field labels are also all uppercased
+        address: "ENDEREÇO",
+        amount: "QUANTIDADE",
+        confirmPassword: "CONFIRMAR PALAVRA-PASSE",
+        daemonLogLevel: "NÍVEL DE LOG PARA O SERVIÇO",
+        daemonP2pPort: "PORTA P2P DO SERVIÇO",
+        daemonZMQPort: "PORTA ZMQ DO SERVIÇO",
+        dataStoragePath: "DIRECTÓRIO DE ARMAZENAMENTO DOS DADOS",
+        filterTransactionType: "FILTRAR POR TIPO DE TRANSAÇÃO",
+        filterTxId: "FILTRAR POR TXID",
+        internalWalletPort: "PORTA INTERNA DA CARTEIRA",
+        keyImages: {
+            exportDirectory: "DIRECTORIA DE EXPORTAÇÃO DAS CHAVES DE IMÁGEM",
+            importFile: "FICHEIRO DE IMPORTAÇÃO DAS CHAVES DE IMÁGEM"
+        },
+        limitDownloadRate: "LIMITAR VELOCIDADE DE DOWNLOAD",
+        limitUploadRate: "LIMITE DA VELOCIDADE DE UPLOAD",
+        localDaemonIP: "IP LOCAL DO SERVIÇO",
+        localDaemonPort: "PORTA LOCALA DO SERVIÇO",
+        maxIncomingPeers: "NÚM. MAX DE PARES DE ENTRADA",
+        maxOutgoingPeers: "NUM. MAX DE PARES DE SAÍDA",
+        mnemonicSeed: "SEMENTE MNEMÓNICA",
+        name: "NOME",
+        newWalletName: "NOME DA NOVA CARTEIRA",
+        notes: "NOTAS",
+        optional: "OPCIONAL",
+        password: "PALAVRA-PASSE",
+        paymentId: "ID DE PAGAMENTO",
+        priority: "PRIORIDADE",
+        remoteNodeHost: "NÓDULO HÓSPEDE REMOTO",
+        remoteNodePort: "PORTA REMOTA DO NÓDULO",
+        restoreFromBlockHeight: "RESTAURAE DESDE ALTURA DO BLOCO",
+        restoreFromDate: "RESTAURAR DESDE DATA",
+        seedLanguage: "LINGUA DA SEMENTE",
+        serviceNodeCommand: "COMANDO DO NÓDULO DE SERVIÇO",
+        serviceNodeKey: "CHAVE DO NÓDULO DE SERVIÇO",
+        walletFile: "FICHEIRO DA CARTEIRA",
+        walletLogLevel: "NIVEL DE LOG DA CARTEIRA",
+        walletName: "NOME DA CARTEIRA",
+        walletRPCPort: "PORTA RPC DACARTEIRA",
+        walletStoragePath: "DIRECTORIA DE SALVAMENTO DA CARTEIRA",
+
+        // These are specific labels which do not get uppercased
+        confirmNewPassword: "Confirmar Nova Palavra-Passe",
+        newPassword: "Nova Palavra-Passe",
+        oldPassword: "Antiga Palavra-Passe",
+        rescanFullBlockchain: "Re-examinar o blockchain completo",
+        rescanSpentOutputs: "Re-examinar saídas gastas",
+        transactionNotes: "Notas de Transações",
+        chooseNetwork: "Seleccione uma Rede",
+        network: "Rede"
+    },
+    footer: {
+        ready: "Pronto",
+        scanning: "Examinando",
```

Comment:
scanning: "Escaneando",

Reply:
Here, I don't agree, as per discussion above. It doesn't become neutral Portuguese, as "escaneando" in Portuguese formally means to digitise something, and not "to search":
https://www.linguee.pt/ingles-portugues/traducao/scanning.html

Also, I would need to change all the "reexaminar" as it would clash in consistency.